cheat.lua

```diff
@@ -1,3 +1,22 @@
+function HandleSocialSpyCommand(Split, Player)
+	if Split[2] == nil then
+		helperToggleSocialSpy(Player)
+	else
+		cRoot:FindAndDoWithPlayer(Split[2], helperToggleSocialSpy)
```

Comment:
However FindAndDoWithPlayer does not work with the Split... (on my server with only me online)